libsubid/api.c

```diff
@@ -81,7 +81,7 @@ void subid_free_ranges(struct subordinate_range **ranges, int count)
 	return free_subordinate_ranges(ranges, count);
 }
 
-int get_subid_owner(unsigned long id, uid_t **owner, enum subid_type id_type)
+int get_subid_owner(unsigned long id, enum subid_type id_type, uid_t **owner)
```

Comment:
This (and few other internal functions) could be `static`, I think? (as `get_subid_ranges()` is)